src/main/java/com/oya/kr/global/exception/GlobalErrorCode.java

```diff
@@ -0,0 +1,29 @@
+package com.oya.kr.global.exception;
+
+import org.springframework.http.HttpStatus;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public enum GlobalErrorCode implements ErrorCode {
+	ERROR_TEST("S001", HttpStatus.BAD_REQUEST, "테스트 에러입니다.");
```

Comment:
```suggestion

	ERROR_TEST("S001", HttpStatus.BAD_REQUEST, "테스트 에러입니다."),
	;
```
enum 에는 끝에 쉼표가 있어도 에러가 안납니당 그래서 저는 enum 타입 코드를 빠르게 추가하기 위해 다음과 같이 세미콜론을 아래를 두는 편이에요!